client/cl_main.lua

```diff
@@ -0,0 +1,316 @@
+    local accountName = QBX.PlayerData[group].name
+    local employees = lib.callback.await('qbx_management:server:getemployees', false, accountName, group)
+    for _, v in pairs(employees) do
+        employeesMenu[#employeesMenu + 1] = {
+            title = v.name,
+            description = v.grade.name,
+            event = 'qbx_management:client:ManageEmployee',
+            args = {
+                group = group,
+                player = v,
+                work = QBX.PlayerData[group]
+            }
+        }
+    end
+
+    employeesMenu[#employeesMenu + 1] = {
+        title = 'Return',
+        icon = 'fa-solid fa-angle-left',
+        event = 'qbx_management:client:openMenu',
+        args = group
+    }
+
+    lib.registerContext({
+        id = 'qbx_management_open_Manage',
+        title = group == 'gang' and 'Manage Gang Members - ' .. string.upper(QBX.PlayerData.gang.label) or 'Manage Employees - ' .. string.upper(QBX.PlayerData.job.label),
+        options = employeesMenu
+    })
+
+    lib.showContext('qbx_management_open_Manage')
+end)
+
+---@param data {source: number, player: table, work: table, group: 'job'|'gang'}
+RegisterNetEvent('qbx_management:client:ManageEmployee', function(data)
+    local employeeMenu = {}
+    local employeeLoop = data.group == 'gang' and gangs[data.work.name].grades or jobs[data.work.name].grades
+    for k, v in pairs(employeeLoop) do
+        employeeMenu[#employeeMenu + 1] = {
+            title = v.name,
+            description = 'Grade: ' .. k,
+            serverEvent = 'qbx_management:server:updateGrade',
+            args = {
+                cid = data.player.empSource,
+                grade = tonumber(k),
+                gradename = v.name,
+                group = data.group
+            }
+        }
+    end
+
+    employeeMenu[#employeeMenu + 1] = {
+        title = data.group == 'gang' and 'Expel Gang Member' or 'Fire Employee',
+        icon = 'fa-solid fa-user-large-slash',
+        serverEvent = 'qbx_management:server:fireEmployee',
+        args = {
+            source = data.player.empSource,
+            group = data.group
+        }
+    }
+
+    employeeMenu[#employeeMenu + 1] = {
+        title = 'Return',
+        icon = 'fa-solid fa-angle-left',
+        event = 'qbx_management:client:openMenu',
+        args = data.group
+    }
+
+    lib.registerContext({
+        id = 'qbx_management_open_Member',
+        title = data.group == 'gang' and 'Manage ' .. data.player.name .. ' - ' .. string.upper(QBX.PlayerData.gang.label) or 'Manage ' .. data.player.name .. ' - ' .. string.upper(QBX.PlayerData.job.label),
+        options = employeeMenu
+    })
+
+    lib.showContext('qbx_management_open_Member')
+end)
+
+---@param group 'job'|'gang'
+RegisterNetEvent('qbx_management:client:hireMenu', function(group)
+    local hireMenu = {}
+    local players = findPlayers()
+    local hireName = QBX.PlayerData[group].name
+    for _, v in pairs(players) do
```

Comment:
Consider a more descriptive variable name than 'v'
```suggestion
    for _, player in pairs(players) do
```